applications/webapp/src/index.tsx

```diff
@@ -41,7 +41,10 @@ initKeycloak()
     setInterval(refreshToken, intervalTime);
   })
   .catch((error) => {
-    console.log(error);
+    console.log(
+      `[index.initKeycloak()] An error occurred during keycloak initialization:`,
+      error,
+    );
   });
```

Comment:
You have to handle the error like `throw new Error("error your message")`. don't log error messages to the console